lib/integrations/awsoidc/idp_iam_config.go

```diff
@@ -180,23 +180,24 @@ func ConfigureIdPIAM(ctx context.Context, clt IdPIAMConfigureClient, req IdPIAMC
 	}
 	log.Printf("IAM OpenID Connect Provider created: url=%q arn=%q.", req.ProxyPublicAddress, aws.ToString(createOIDCResp.OpenIDConnectProviderArn))
 
-	if err := createIdPIAMRole(ctx, clt, req); err != nil {
+	createdIdpIAMRoleArn, err := createIdPIAMRole(ctx, clt, req)
+	if err != nil {
 		return trace.Wrap(err)
 	}
-	log.Printf("IAM Role %q created.", req.IntegrationRole)
+	log.Printf("IAM Role created: name=%q arn=%q", req.IntegrationRole, aws.ToString(createdIdpIAMRoleArn))
```

Comment:
If we could enable a way for them to copy the entire string with a double or triple click it would be great.
From someone, who usually tries that instead selecting the required text (and usually misses by a char or two)

Reply:
i think we can just make this simpler by just telling the user to go to their role dashboard (they have a copy and paste button):

<img width="183" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/90891948-624c-4803-9925-80ee252a2123">

simplifies to:

<img width="1066" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/32ba1b13-97fb-4066-9562-53564a64e664">

and i'll leave the CLI output as is (since it matches the arn output for oidc provider)

Reply:
Looks good :+1: